app/assets/javascripts/qsets.js

```diff
@@ -41,6 +41,7 @@ function initQuestionDisplayModal($modal, $question_link) {
 
   initAnswerButton();
   initUserFeedback();
+  initVote();
```

Comment:
Now that users will be using the modal only when they select the "Quiz All" option, it is no longer necessary to allow them to vote on individual questions in the modal and I no longer need to implement that functionality in this PR. They can still access and vote on individual questions from the question-specific page. Thanks for catching that, though!